helium/src/scss/_forms.scss

```diff
@@ -87,37 +112,48 @@ $FIELD_MAX_WIDTH: 800px;
 
   .email, .username {
     label {
-      @include replace-text('mail_icon.png');
+      @include yc-left-icon('fa/envelope');
     }
   }
 
   .password, .password1, .password2 {
     label {
-      @include replace-text('password.png');
+      @include yc-left-icon('fa/eye');
```

Comment:
This should be `fa/lock` for the design to be consistent <http://fontawesome.io/icon/lock/>